app/src/main/res/layout/content_item.xml

```diff
@@ -14,6 +14,22 @@
       type="org.oppia.android.app.player.state.itemviewmodel.ContentViewModel" />
   </data>
 
+  <!-- EXPLORATION SPLIT VIEW -->
+  <!-- Overall Outer Space: 24, 40, 40, 0 -->
+  <!-- Overall Inner Space: 12, 16, 12, 16 -->
+
+  <!-- EXPLORATION VIEW -->
+  <!-- Overall Outer Space: 24, 24, 40, 0 -->
+  <!-- Overall Inner Space: 8, 12, 8, 12 -->
+
+  <!-- QUESTION SPLIT VIEW -->
+  <!-- Overall Outer Space: 32, 40, 32, 12 -->
+  <!-- Overall Inner Space: 0, 0, 0, 0 -->
+
+  <!-- QUESTION VIEW -->
+  <!-- Overall Outer Space: 32, 36, 32, 12 -->
+  <!-- Overall Inner Space: 00, 00, 00, 00 -->
```

Comment:
a single zero will be good as the above one

Reply:
Done.

Reply:
Resolving as @chrk2205 cannot resolve this comment.